content/blog/2018-12-11-rcites.md

```diff
@@ -0,0 +1,83 @@
+    twitter: KCazelles
+  - name: Jonas Geschke
+    url:  https://www.linkedin.com/in/jonasgeschke/
+    twitter: J_Geschke
+date: 2018-12-11
+categories: blog
+topicid: XXX 
+Tags:
+- r
+- package
+- data-access
+- API-client
+- cites 
+---
+
+### The Ecology Hackathon
+
+Almost one year ago now, ecologists filled a room to meet for an “Ecology Hackathon: Developing R Packages for Accessing, Synthesizing and Analyzing Ecological Data” that was part of the BES, GfÖ, NecoV, and EEF Joint Annual Meeting 2017 “Ecology Across Borders” in Ghent, co-organised by rOpenSci and Methods in Ecology and Evolution (see [Methodsblog entry 1](https://methodsblog.com/2017/11/16/hackathon-challenges/) and [Methodsblog entry 2](https://methodsblog.com/2018/01/23/ecology-hackathon/). At different tables, different people joined each other to work on different ideas to implement as R packages. At our table, we were around ten people that more or less did not know anything about what we aimed for. We barely knew each other and nobody had clear expectations, just the desire of learning more about R packages. As a piece of paper on our table indicated, we were interested in a common idea posted as a [wishlist](https://github.com/ropensci/wishlist/issues/29) in the rOpenSci community: building an R package to interact with [CITES](https://cites.org/) and its [Speciesplus database](https://speciesplus.net). At 10 am, nobody had a clear idea on where to start. By 6 pm, we had a functional prototype of the rcites package, which was really rewarding and gave motivation to follow up on the package development. We did a great team-work, met new researchers, learn a bunch of new stuff, this was definitely a successful hackathon!   
+
+Now, almost one year after, it appears to us that there were several key issues that the hackathon turned to be a success. The first was the short but thorough introduction to the development of R packages by Maëlle Salmon (Maëlle became an editor at rOpenSci shortly after). Within an hour or so we had a clear vision of what the development of an R package pipeline looks like. A second key was the help of more experienced people. Our group immensely benefited from the experience of [Tamora James](https://twitter.com/soayipm) that kindly answered our questions. The last one was the diversity of people interested in the project. Indeed, while some of us were beginner R users, others had moderate experience in creating packages and others had already dealt with CITES before, so we were able to split tasks organically. Some of us learned how to access the CITES/Speciesplus API (https://api.speciesplus.net/) and how to write a request via URLs. Others worked on rough versions of a functional code, yet again others started writing the help files and tested the package. After a few hours of frustration, ignoring why a given function refused to work as intended, things started to fit like pieces of a jigsaw puzzle. This was very empowering.   
```

Comment:
CITES/Speciesplus API (https://api.speciesplus.net/) -> `[CITES/Speciesplus API](https://api.speciesplus.net/)` so that the text is linked, rather than displaying the URL